src/WireMock.Net.OpenApiParser/Utils/ExampleValueGenerator.cs

```diff
@@ -28,38 +29,55 @@ public object GetExampleValue(OpenApiSchema schema)
             switch (schema?.GetSchemaType())
             {
                 case SchemaType.Boolean:
-                    return _settings.ExampleValues.Boolean;
+                    OpenApiBoolean exampleBoolean = (OpenApiBoolean)schema?.Example;
```

Comment:
if you case to something (like OpenApiBoolean), the result cannot be null I think?

Not 100% sure....

Maybe I need to introduce NUllable and langversion 9+ in these projects to get better null checks..

Reply:
Yes you right, I changed the null validation to `exampleBoolean` because it could be null, not the value but the variable itself.